lib/heroku/command/domains.rb

```diff
@@ -13,17 +14,35 @@ class Domains < Base
     #Examples:
     #
     # $ heroku domains
-    # === Domain names for example
-    # example.com
+    # === Development Domain
+    # example.herokuapp.com
+    #
+    # === Custom Domains
+    # Domain Name  CNAME Target
```

Comment:
there was a lot of discussion about the wording for this column in Dashboard, and this was copied from there. @tim-lang, @itchymutt: thoughts? I don't think we should block on this, but worth consideration for a follow up.